sagemaker-pipelines/step-decorator/computer-vision-examples/computer-vision-pipeline.ipynb

```diff
@@ -0,0 +1,1101 @@
+{
```

Comment:
<blockquote>However, <code>source-ref</code> needs to be the S3 location of the image</blockquote><p> </p><p>Is it because we upload the dataset to s3 and the pipeline retrieves data from s3?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
I will clarify within the notebook why we need to update `source-ref` to the S3 URIs.